src/test/java/org/folio/consortia/controller/SelfControllerTest.java

```diff
@@ -0,0 +1,81 @@
+package org.folio.consortia.controller;
+
+import org.folio.consortia.domain.dto.UserTenant;
+import org.folio.consortia.domain.dto.UserTenantCollection;
+import org.folio.consortia.repository.ConsortiumRepository;
+import org.folio.consortia.service.UserTenantService;
+import org.folio.consortia.support.BaseTest;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.integration.XOkapiHeaders;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class SelfControllerTest extends BaseTest {
+
+  @Autowired
+  FolioExecutionContext folioExecutionContext;
+  @MockBean
+  ConsortiumRepository consortiumRepository;
+  @MockBean
+  UserTenantService userTenantService;
+  private static final String CONSORTIUM_ID = "7698e46-c3e3-11ed-afa1-0242ac120002";
+
+  @Test
+  void shouldGetAssociatedAffliations() throws Exception {
+    List<UserTenant> userTenantDtos = List.of(new UserTenant(), new UserTenant());
+    UserTenantCollection userTenantCollection = new UserTenantCollection();
+    userTenantCollection.setUserTenants(userTenantDtos);
+    userTenantCollection.setTotalRecords(userTenantDtos.size());
+
+    var headers = defaultHeaders();
+    UUID consortiumId = UUID.fromString(CONSORTIUM_ID);
+    when(consortiumRepository.existsById(consortiumId)).thenReturn(true);
+    when(userTenantService.getByUserId(any(),any(),any(),any())).thenReturn(userTenantCollection);
+    this.mockMvc.perform(
+      get("/consortia/7698e46-c3e3-11ed-afa1-0242ac120002/_self")
+        .headers(headers)
+      )
+      .andExpectAll(status().isOk());
```

Comment:
please add check to this method that tenant collection returned, not only status